### ISSUE TITLE
Incorrect behavior when opening menu skype in the system tray.

### ISSUE THREAD

**Issue by jawhien** [nvaccess/nvda#8705]
If you open the menu button skype in the system tray, NVDA behaves incorrectly.

### Steps to reproduce:
Start the skype program. Go to the system tray by pressing "windows + b". Find the skype button to open the context menu.

### Actual behavior:
On the menus impossible to navigate use the arrow keys.
Menu can't close using the Escape key or ALT + F4.

### Expected behavior:
The arrow keys should move through the menu selections. The menu should be closed using the Escape key or ALT + F4.

#### NVDA Installed/portable/running from source:
Installed and portable.
 
#### NVDA version:
2018.2.1

#### Windows version:
Windows 7 Home Premium X64.

#### Name and version of other software in use when reproducing the issue:
Skype 8.29.0.47

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA?
2018.3 beta3, there is a problem.

**Comment by jscholes** [comment]
Could you provide a speech log, via either the NVDA Speech Viewer or via setting the log level to input/output in NVDA's general settings dialog?  I suspect this is a problem with Skype rather than NVDA; now that it's using Electron the menu will probably be rendered using Chrome rather than as a native control, and these menus being inaccessible is very common.

**Comment by jawhien** [comment]
I forgot to write that the menu items are available with the keys NVDA + numpade2, then NVDA + numpade4 or NVDA + numpade6.
Here is the log, hope this is what you need.

[log.txt](https://github.com/nvaccess/nvda/files/2342739/log.txt)

**Comment by florianionascu7** [comment]
Yes, I can confirm the issue. I'm using Windows 10 version 1803 - build 17134, Skype 8.29 and NVDA 2018.2.1.

**Comment by Adriani90** [comment]
@florianionascu7, @jawhien is this still an issue with NVDA 2018.4?

**Comment by lukaszgo1** [comment]
I can confirm this with Windows 7 pro and Skype 8.34

**Comment by Adriani90** [comment]
This is still reproducible in Skype Electron for desktop 8.43 and NVDA 2019.1.1. Here is the developer info:

```
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (22:46:46.463):
Developer info for navigator object:
name: u''
role: ROLE_POPUPMENU
states: STATE_HASPOPUP
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.IAccessible.ia2Web.Ia2Web object at 0x04983E70>
Python class mro: (<class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: RectLTWH(left=2442, top=1357, width=393, height=286)
value: u''
appModule: <'skype' (appName u'skype', process ID 2348) at address 50e4070>
appModule.productName: u'Skype'
appModule.productVersion: u'8.43'
TextInfo: <class 'NVDAObjects.IAccessible.IA2TextTextInfo'>
windowHandle: 591968L
windowClassName: u'Chrome_WidgetWin_2'
windowControlID: 0
windowStyle: -1778384896
extendedWindowStyle: 134217736
windowThreadID: 5264
windowText: u''
displayText: u''
IAccessibleObject: <POINTER(IAccessible2) ptr=0x689a674 at 5219b20>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=591968, objectID=-4, childID=-4343
IAccessible accName: u''
IAccessible accRole: ROLE_SYSTEM_MENUPOPUP
IAccessible accState: STATE_SYSTEM_HASPOPUP, STATE_SYSTEM_VALID (1073741824)
IAccessible accDescription: None
IAccessible accValue: u''
IAccessible2 windowHandle: 591968
IAccessible2 uniqueID: -4343
IAccessible2 role: ROLE_SYSTEM_MENUPOPUP
IAccessible2 states: IA2_STATE_VERTICAL, IA2_STATE_OPAQUE (132096)
IAccessible2 attributes: u'class:SubmenuView;'
```

**Comment by Adriani90** [comment]
Still reproducible with NVDA 2019.3 Beta 1 and Skype electron 8.55.

**Comment by k-kolev1985** [comment]
Actually, this was fixed maybe a few months ago, but it started occurring again in Skype v8.55. So, it is a regression. I've filed a bug report in the Windows 10 Feedback Hub. So, anyone that can reproduce it, may go and approve/upvote the report there. Here is the link: [The context menu for the system tray icon of Skype 8.55 is not accessible for screen readers](https://aka.ms/AA6vq6k).

**Comment by Adriani90** [comment]
It seems fixed in Skype 94 and NVDA 2023.1 RC1. I tested both with Electron and UWP. I am closing this as works for me. Please comment in case you are still having this issue and we can reopen.
